readme.md

```diff
@@ -80,15 +80,15 @@ Once installation is working, take a look at the existing code to make sure you
 
 * **Question:** This code is currently _very_ insecure. Why?
 
-* **Your Answer:**
+* **Your Answer:** not encrypting the password, no checking for duplicate usernames
 
 * **Question:** What would happen if three different users tried to sign up with the same username? How can we prevent that?
 
-* **Your Answer:**
+* **Your Answer:** they could potentially see other peoples data, we could check the database for existing usernames
 
 * **Question:** Why are we making our route `POST /api/signup` as opposed to `POST /api/users`?
 
-* **Your Answer:**
+* **Your Answer:** it's more official - gives the user an idea of the action they are taking. 
```

Comment:
Yes, but also the process of signing up is also slightly different than just creating a user. There could be an instance where you have both, for example.
1. One for users to sign up
1. One for admin to create new users